### PR TITLE
Avoid logging through mozlog in tests experiment

### PR DESCRIPTION
**Pull request by EnTeQuAk**
Closes https://github.com/mozilla/addons-server/issues/8451 (Doesn't necessary fix it, we don't want a unified logging-level but this PR makes logging much clearer.)

### CODE REVIEW

File: src/olympia/conf/stage/settings.py [hunk 1]
```diff
@@ -103,6 +103,10 @@
     'z.pool': {'level': logging.ERROR},
 })
 
+# Update the logger name used for mozlog
+LOGGING['formatters']['json']['logger_name'] = 'http_app_addons_stage'
```
Comment (eviljeff) [inline src/olympia/conf/stage/settings.py]: I expect you did it deliberately but calling out this is a change - previously stage didn't override the name so used the same as prod used.

Comment (EnTeQuAk) [reply src/olympia/conf/stage/settings.py]: it does set MOZLOG_NAME = "http_app_addons_stage", I simply forgot to remove it. I'll update the patch Monday morning

Comment (eviljeff) [reply src/olympia/conf/stage/settings.py]: ah

Comment (EnTeQuAk) [reply src/olympia/conf/stage/settings.py]: Fixed in 3b77750793

**Pull request merged**